test/test-util.js

```diff
@@ -121,8 +121,13 @@ function testStateTransitions(ns, initial, newState, finalState, incNoDelta, sta
 function prepareCluster(insert_fns) {
     return function(t, tc, n) {
         return [
-            dsl.assertDetectChecksumMethod(t, tc),
+            // By waiting for the first ping we make sure the SUT is ready to go.
+            // We don't consume it so other tests (especially the ping-tests)
+            // can still assert implementation details related to ping distribution.
+            dsl.waitForPing(t, tc, false),
```

Comment:
After a discussion with Nils we changed it back (and added a comment). The reason why we're waiting for the initial ping first is so we wait until the node is fully ready before checking the number of joins. This way we're able to detect if there are too many joins during standup.